examples/demo/multistroke/settings.py

```diff
@@ -95,4 +95,3 @@ def on_touch_down(self, touch):
 Factory.register('MultistrokeSettingBoolean', cls=MultistrokeSettingBoolean)
 Factory.register('MultistrokeSettingSlider', cls=MultistrokeSettingSlider)
 Factory.register('MultistrokeSettingString', cls=MultistrokeSettingString)
-
```

Comment:
make style was showing  error to it.
i rechecked it.i purposely put a blank line at the end it showed "./examples/demo/multistroke/settings.py:98:1:   W391 blank line at end of file
"
that is why i removed it.